src/Silex/ExceptionListenerWrapper.php

```diff
@@ -77,13 +78,15 @@ protected function shouldRun(\Exception $exception)
         return true;
     }
 
-    protected function ensureResponse($response, GetResponseForExceptionEvent $event)
+    protected function ensureResponse($response, ExceptionEvent $event)
     {
         if ($response instanceof Response) {
             $event->setResponse($response);
         } else {
-            $viewEvent = new GetResponseForControllerResultEvent($this->app['kernel'], $event->getRequest(), $event->getRequestType(), $response);
```

Comment:
https://github.com/symfony/http-kernel/blob/v4.4.50/Event/GetResponseForControllerResultEvent.php

```
/**
 * @deprecated since Symfony 4.3, use ViewEvent instead
 */
```